packages/nimble-components/src/utilities/wait-until-custom-elements-defined-async.ts

```diff
@@ -1,8 +1,10 @@
 export async function waitUntilCustomElementsDefinedAsync(
     elements: Element[]
 ): Promise<void> {
-    const definedElements = elements.map(async item => (item.matches(':not(:defined)')
-        ? customElements.whenDefined(item.localName)
-        : Promise.resolve()));
+    const definedElements = elements.map(async item => {
+        if (item.matches(':not(:defined)')) {
+            await customElements.whenDefined(item.localName);
+        }
+    });
```

Comment:
This was the only other interesting change, The fixer resulted in:

```ts
    const definedElements = elements.map(async item => (item.matches(':not(:defined)')
        ? await customElements.whenDefined(item.localName)
        : await Promise.resolve()));
```

And that's technically fine but it looked funny. We don't actually care about this function returning anything, it just needs to resolve when everything is defined. I tweaked the function and I think that is conveyed more clearly now.

Reply:
In retrospect doing a filter and only awaiting what needs to be awaited would have been even better, but it's a nit:

```ts
const definedElements = elements
    .filter(item => item.matches(':not(:defined)'))
    .map(async item => {
        await customElements.whenDefined(item.localName);
    });
```